pom.xml

```diff
@@ -175,7 +175,7 @@
         <dependency>
           <groupId>com.fasterxml.jackson.core</groupId>
           <artifactId>jackson-databind</artifactId>
-          <version>2.9.8</version>
+          <version>2.9.10.5</version>
```

Comment:
I packaged/installed a new MCD version using this jackson-databind version and it works just the same.

I think we are ok with this change.